invenio_administration/assets/semantic-ui/js/invenio_administration/src/search/SearchResults.js

```diff
@@ -33,9 +34,7 @@ export const SearchResults = ({ paginationOptions, currentResultsState }) => {
           <Grid.Column width={4}>
             <Count
               label={() => (
-                <>
-                  {total} {i18next.t("result(s) found")}
-                </>
+                <>{i18next.t("{{count}} results found", { count: total })}</>
```

Comment:
can we use `Trans` here?

Reply:
Unless there is some react/html in the string, we can use simple `t` function. See here: https://react.i18next.com/latest/trans-component#important-note

Reply:
So the difference with below is that in line 54 we need the html key and that's why we use `Trans`. Right?